ams/system.py

```
@@ -776,7 +777,9 @@ def to_m(self, outfile: str, overwrite: bool = None):
         """
         return write_m(self, outfile=outfile, overwrite=overwrite)
 
-    def to_xlsx(self, outfile: str, overwrite: bool = None):
+    def to_xlsx(self, outfile: str, overwrite: bool = None,
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)